src/components/FeedView.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { RefreshableFeed } from './RefreshableFeed';
+import { Feed } from '../models/Feed';
+import { Post } from '../models/Post';
+import { Settings } from '../models/Settings';
+import { NavigationHeader } from './NavigationHeader';
+import { Colors } from '../styles';
+
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import * as AreYouSureDialog from './AreYouSureDialog';
+
+export interface DispatchProps {
+    onRefreshPosts: (feeds: Feed[]) => void;
+    onFollowFeed: (feed: Feed) => void;
+    onUnfollowFeed: (feed: Feed) => void;
+    onToggleFavorite: (feedUrl: string) => void;
+}
+
+export interface StateProps {
+    navigation: any;
+    uri: string;
```

Comment:
`uri` is not really a good name, because of two reasons:
- It does not tell what does it belong to (I guess it's the feed uri, but I am not sure)
- we should not use `uri` anywhere in the code, because it's meaning is too broad. I know it's already used at other places, but we should change those as well. The best would be to use a different type for this, but then we should use a smarter JSON serializing/deserializing library than the built-in ones.

Instead we can just use url everywhere, in this case `feedUrl`.

Reply:
good point

Reply:
I renamed it in the navParams as well